software/siyuan.yml

```
@@ -0,0 +1,11 @@
+name: "SiYuan"
+website_url: "https://b3log.org/siyuan/"
+source_code_url: "https://github.com/siyuan-note/siyuan"
+description: "A privacy-first, self-hosted, fully open source personal knowledge management software, written in typescript and golang."
```

```suggestion
description: "A privacy-first personal knowledge management software, written in typescript and golang."
```
Per https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines, minimized description to convey the software better. Feel free to improve mine other collaborators, or if you think this is good. 👍🏻

---

@TornaxO7 ping

---

Sorry for the late action from my side! Suggestion applied :+1: